sdk/search/ai-search/README.md

```diff
@@ -0,0 +1,302 @@
+- Manage indexers that pull data from a data source into an index.
+- Query documents in the index with a powerful set of search APIs that support faceted navigation, typeahead queries, suggestions, and geo-spatial search.
+- Enrich your search index with AI skills that add structure or extract meaning from raw documents during indexing.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/search/ai-search/) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/ai-search) |
+[API reference documentation](https://aka.ms/azsdk-js-search-ref-docs) |
+[Product documentation](https://docs.microsoft.com/en-us/azure/search/) |
+[Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/search/ai-search/samples)
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 8.x.x or higher
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- An existing [Cognitive Search][search_resource] resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
+
+If you use the Azure CLI, replace `<your-resource-group-name>` and `<your-resource-name>` with your own unique names:
+
+```PowerShell
+az search service create --resource-group <your-resource-group-name> --name <your-resource-name> --sku S
+```
+
+The above creates a resource with the "Standard" pricing tier. See [choosing a pricing tier](https://docs.microsoft.com/en-us/azure/search/search-sku-tier) for more information.
+
+### 1. Install the `@azure/ai-search` package
+
+```bash
+npm install @azure/ai-search
+```
+
+### 2. Create and authenticate a `SearchIndexClient`
+
+Cognitive Search uses keys for authentication.
+
+#### Using an Admin Key
+
+Use the [Azure CLI][azure_cli] snippet below to get the Admin Key from the Cognitive Search resource.
+
+```PowerShell
+az search admin-key show --resource-group <your-resource-group-name> --service-name <your-resource-name>
+```
+
+Alternatively, you can get the endpoint and Admin Key from the resource information in the [Azure Portal][azure_portal].
+
+Once you have an Admin Key, you can use it as follows:
+
+```js
+const { SearchIndexClient, SearchApiKeyCredential } = require("@azure/ai-search");
+
+const client = new SearchIndexClient(
+  "<endpoint>",
+  "<indexName>",
+  new SearchApiKeyCredential("<Admin Key>");
+);
+```
+
+## Key concepts
+
+### SearchIndexClient
+
+`SearchIndexClient` is one of the primary interface for developers using the Cognitive Search client library. It provides asynchronous methods for working with documents in an index.
+
+## Examples
+
+### Query documents in an index
+
+To list all results of a particular query, you can use `listSearchResults` with a search string that uses [simple query syntax](https://docs.microsoft.com/en-us/azure/search/query-simple-syntax):
+
+```js
+for await (const result of client.listSearchResults("wifi -luxury")) {
+  console.log(result);
```

Comment:
Does `result` actually toString() nicely? Or are there a couple of fields worth showcasing in the example?

Reply:
it's a plain JSON object that has the same keys as your schema, so it's actually pretty decent (at least in Node)